ocsp/src/main/java/io/nats/ocsp/OcspExample.java

```diff
@@ -54,10 +45,11 @@ public static SSLContext getVmWideContext() throws NoSuchAlgorithmException, Key
                 PemUtils.loadTrustMaterial(ocspCaCertPath());
 
         // Construct and initialize the sslContext
-        SSLContext sslContext = SSLContext.getInstance("TLSv1.2");
-        sslContext.init(new KeyManager[]{keyManager}, new TrustManager[]{trustManager}, new SecureRandom());
-
-        return sslContext;
+        return SSLFactory.builder()
+                .withIdentityMaterial(keyManager)
+                .withTrustMaterial(trustManager)
+                .build()
+                .getSslContext();
```

Comment:
I know this simplifies it, and I would use it myself. But for someone who is not using the utils, I think it's useful to show how to make the context.

Reply:
Ah, that explains it. I will revert it.